### PR TITLE
Correct JS Flows example #3

### PR DESCRIPTION
**Pull request by inlined**
Example is for structured output but the schema was string.

Checklist (if applicable):
- [ ] Tested (manually, unit tested, etc.)
- [ ] Changelog updated
- [x] Docs updated

### CODE REVIEW

**Comment by kevinthecheung** [comment]
This one really does return a string though

**Comment by kevinthecheung** [comment]
In the documentation, this example is used to show that the output schema of a generate call might not be the same as the output schema of the flow it’s in

File: js/doc-snippets/src/flows/index.ts [hunk 1]
```diff
@@ -67,7 +67,10 @@ export const menuSuggestionFlowMarkdown = ai.defineFlow(
   {
     name: 'menuSuggestionFlow',
     inputSchema: z.string(),
-    outputSchema: z.string(),
+    outputSchema: z.object({
```
Comment (pavelgj) [inline js/doc-snippets/src/flows/index.ts:70]: No... the flow output type is `z.string()`... see line 84:
```ts
return `**${output.dishname}**: ${output.description}`
```

Comment (inlined) [reply js/doc-snippets/src/flows/index.ts:70]: Oh whoops. Dunno how I got confused when reading docs.

**Pull request closed without merging**